examples/with-universal-configuration-runtime-docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:9-slim AS builder
+WORKDIR /app
+COPY package.json .
+RUN npm install
```

Comment:
Lets use yarn, it's included in `mhart/alpine-node` by default, so no need to install it. Just:

`RUN yarn install`

Reply:
Lets not do multi-stage builds right now, but just `EXPOSE 3000` `CMD yarn start` in this step.

The reason for this is that `now` doesn't support multi-stage yet, it's coming soon though.

Reply:
Actually, `now` do support multi-stage build in OSS.

Reply:
I put this information in README.